docs/v3/how-to-guides/cloud/manage-users/configure-sso.mdx

```diff
@@ -47,5 +47,54 @@ Prefect Cloud resources. This allows your IT team to control access to Prefect C
 
 ## SCIM Provisioning
 
-Enterprise plans have access to SCIM for user provisioning.
-The SSO tab provides access to enable SCIM provisioning.
+Enterprise plans have access to SCIM (System for Cross-domain Identity Management) for automated user and team provisioning.
+
+SCIM enables automatic provisioning and de-provisioning of users and teams based on your identity provider's directory.
+When users are added, removed, or have their group memberships changed in your identity provider, these changes are
+automatically reflected in Prefect Cloud.
+
+### Enable SCIM provisioning
+
+The SSO tab provides access to enable SCIM provisioning for your account.
+
+### Role mapping
+
+SCIM supports role mapping, which allows you to automatically assign Prefect Cloud account roles based on directory
+group memberships configured in your identity provider.
+
+#### Account roles
+
+Directory groups can be mapped to the following Prefect Cloud account roles:
+
+- **Owner**: Full account control including billing and account management
+- **Admin**: Account management capabilities, can invite users and manage workspaces
+- **Member**: Basic account access with workspace-level permissions
+
+#### Configure role mapping
+
+Role mapping is configured in your WorkOS organization dashboard:
+
+1. Navigate to your WorkOS dashboard
+2. Select your organization
```

Comment:
I _believe_ it'll go directly to their organization when they go through the link.